code/datums/elements/food/processable.dm

```diff
@@ -40,8 +40,7 @@
 		var/found_table = locate(/obj/structure/table) in found_location
 		var/found_tray = locate(/obj/item/storage/bag/tray) in found_location
 		if(!found_turf && !istype(found_location, /obj/item/storage/bag/tray) || found_turf && !(found_table || found_tray))
-			to_chat(user, "<span class='notice'>You cannot make that here! You need a table or at least a tray.</span>")
+			to_chat(user, "<span class='notice'>You cannot make [initial(result_atom_type.name)] here! You need a table or at least a tray.</span>")
 			return
 
-	mutable_recipes += list(list(TOOL_PROCESSING_RESULT = result_atom_type, TOOL_PROCESSING_AMOUNT = amount_created, TOOL_PROCESSING_TIME = time_to_process))
-	return COMPONENT_NO_AFTERATTACK
```

Comment:
why is this removed?

Reply:
me when I undocument changes https://github.com/tgstation/tgstation/pull/58983